can/common.h

```diff
@@ -26,7 +26,8 @@ unsigned int honda_checksum(uint32_t address, const std::vector<uint8_t> &d);
 unsigned int toyota_checksum(uint32_t address, const std::vector<uint8_t> &d);
 unsigned int subaru_checksum(uint32_t address, const std::vector<uint8_t> &d);
 unsigned int chrysler_checksum(uint32_t address, const std::vector<uint8_t> &d);
-unsigned int volkswagen_crc(uint32_t address, const std::vector<uint8_t> &d);
+unsigned int volkswagen_mqb_crc(uint32_t address, const std::vector<uint8_t> &d);
+unsigned int volkswagen_pq_checksum(int checksum_start_bit, const std::vector<uint8_t> &d);
```

Comment:
I'd like to keep the signatures for all the checksum functions the same so we can eventually do something nicer than the if-else-spaghetti. think this would be best:
```suggestion
unsigned int volkswagen_pq_checksum(uint32_t address, const Signal &sig, const std::vector<uint8_t> &d);
```

Reply:
Reasonable. It's a lot of touches, so early next week I'll propose a null-effect refactor of the existing code to work as per your suggestion, then once that passes review, rebase this on top.

Reply:
I took a pass at this in #650, see what you think of it.